src/Core/Framework/Adapter/Twig/EntityTemplateLoader.php

```diff
@@ -55,7 +55,14 @@ public function getSourceContext(string $name): Source
 
     public function getCacheKey(string $name): string
     {
-        return $name;
+        $template = $this->findDatabaseTemplate($name);
+
+        if (!$template) {
```

Comment:
Not sure if we do this anywhere, but for these cases a logger.Warn would be cool so it does not go unnoticed if there is ever a bug that would cause the loading to fail

Reply:
IMHO not needed here, we throw in `getSourceContext()` when we don't find the template, that should make us aware if something is wrong